src/send/InviteRewardsCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { notificationInvite } from 'src/images/Images'
+import { inviteRewardsTypeSelector } from 'src/send/selectors'
+import { InviteRewardsType } from 'src/send/types'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+export default function InviteRewardsCard() {
+  const { t } = useTranslation()
+  const inviteRewardsType = useSelector(inviteRewardsTypeSelector)
+
+  const inviteRewardsToTranslationPrefix: Record<InviteRewardsType, string> = {
+    [InviteRewardsType.CUSD]: 'inviteRewardsBannerCUSD',
+    [InviteRewardsType.NFT]: 'inviteRewardsBanner',
+    [InviteRewardsType.NONE]: 'inviteRewardsBanner',
```

Comment:
Fixed in 1b354b05bb8828a3a61c92816fe7f04500b65af8. I left some of the display logic in `src/send/SendSelectRecipient.tsx` as it seems like it would be more performant to avoid attempting to render at all vs attempting to render the component and returning null when inviteRewardsActive is false.

Applied your recommend where if the prefix is null we don't display the card.